### ISSUE TITLE
Crash Onload of Example

### ISSUE THREAD

**Issue by indivisable** [edwardvalentini/EVContactsPicker#7]
Tried the example but it just crashes with this message:

Works on Simulator but not Actual Device.

```
AddressSanitizer debugger support is active. Memory error breakpoint has been installed and you can now use the 'memory history' command.
hh is Optional("Add Contacts")
2016-12-05 19:11:02.812946 EVContactsPicker_Example[1329:370951] [LayoutConstraints] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
(
    "<NSLayoutConstraint:0x104e1e1d0 UITextView:0x106201c80'No Contacts Selected.'.leading == UIView:0x105811e40.leadingMargin + 84   (active)>",
    "<NSLayoutConstraint:0x104e1e160 UIView:0x105811e40.trailingMargin == UITextView:0x106201c80'No Contacts Selected.'.trailing + 77   (active)>",
    "<NSLayoutConstraint:0x104e1e0f0 UITextView:0x106201c80'No Contacts Selected.'.centerX == UIView:0x105811e40.centerX   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x104e1e160 UIView:0x105811e40.trailingMargin == UITextView:0x106201c80'No Contacts Selected.'.trailing + 77   (active)>

Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKit/UIView.h> may also be helpful.
fatal error: unexpectedly found nil while unwrapping an Optional value
2016-12-05 19:19:53.898333 EVContactsPicker_Example[1329:370951] fatal error: unexpectedly found nil while unwrapping an Optional value
```

**Comment by edwardvalentini** [comment]
i'll take a look - i have made a couple of changes in the last few days its possible i overlooked something.

**Comment by indivisable** [comment]
Commenting out/replacing this line seemed to let it run

 func getContactsFromAddressBook() -> Void {
...
                if(contact.imageDataAvailable) {
                    let imgData = contact.imageData
                    let img = UIImage(data: imgData!) <------------ This throws an error
                    tmpContact.image = img
                } else {
                    tmpContact.image = self.avatarImage
                }
...

**Comment by indivisable** [comment]
@edwardvalentini 

Also it seems there is an issue when you search, this happens on both Simulator and Actual Device. Here is the dump:

```
AddressSanitizer debugger support is active. Memory error breakpoint has been installed and you can now use the 'memory history' command.
hh is Optional("Add Contacts")
2016-12-05 19:53:03.530049 EVContactsPicker_Example[1395:383347] [LayoutConstraints] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
(
    "<NSLayoutConstraint:0x104e1ff90 UITextView:0x106401c80'No Contacts Selected.'.leading == UIView:0x105811e40.leadingMargin + 84   (active)>",
    "<NSLayoutConstraint:0x104e1ff20 UIView:0x105811e40.trailingMargin == UITextView:0x106401c80'No Contacts Selected.'.trailing + 77   (active)>",
    "<NSLayoutConstraint:0x104e1feb0 UITextView:0x106401c80'No Contacts Selected.'.centerX == UIView:0x105811e40.centerX   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x104e1ff20 UIView:0x105811e40.trailingMargin == UITextView:0x106401c80'No Contacts Selected.'.trailing + 77   (active)>

Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKit/UIView.h> may also be helpful.
2016-12-05 19:53:08.518732 EVContactsPicker_Example[1395:383347] [MC] System group container for systemgroup.com.apple.configurationprofiles path is /private/var/containers/Shared/SystemGroup/systemgroup.com.apple.configurationprofiles
2016-12-05 19:53:08.530018 EVContactsPicker_Example[1395:383347] [MC] Reading from public effective user settings.
2016-12-05 19:53:15.165269 EVContactsPicker_Example[1395:383347] *** Terminating app due to uncaught exception 'NSUnknownKeyException', reason: '[<_SwiftValue 0x1097f1f80> valueForUndefinedKey:]: this class is not key value coding-compliant for the key firstName.'
*** First throw call stack:
(0x181b6a1c0 0x1805a455c 0x181b69e88 0x18260e470 0x182555318 0x182597590 0x1825971ac 0x182596f6c 0x182596c90 0x100f9f6d4 0x101074db4 0x100f9ec54 0x100fa8e00 0x100fd2f14 0x100fd3fb0 0x187b8a9c8 0x187a047f8 0x187b8a4c0 0x187b896b0 0x187b89508 0x187b8943c 0x187de52e0 0x187de4e84 0x1879ef26c 0x18265365c 0x181b18278 0x181b17bc0 0x181b157c0 0x181a44048 0x1834ca198 0x187a302fc 0x187a2b034 0x10000de40 0x180a285b8)
libc++abi.dylib: terminating with uncaught exception of type NSException
(lldb) 
```

**Comment by edwardvalentini** [comment]
I've gone ahead and fixed both of these issues and released 0.5.8.      Thanks for letting me know.  I'm going to close this issue

**Comment by indivisable** [comment]
Related https://github.com/edwardvalentini/EVContactsPicker/issues/10
